app/Http/Controllers/Admin/ManagerCrudController.php

```diff
@@ -6,6 +6,8 @@
 
 class ManagerCrudController extends CrudController
```

Comment:
The search function on this page doesn't seem to work. To be fair, I think this is an old issue, so I don't know if you want to fix it in this PR.

Reply:
I can take a quick look